python/lsst/ts/ATDome/mock_controller.py

```diff
@@ -51,17 +52,26 @@ class AzActuator(simactuators.PointToPointActuator):
       For instance a move from 355 deg to 0 deg is a move of 5 deg
       (in the direction of increasing azimuth).
     """
+
     def __init__(self, start_position, speed):
-        super().__init__(min_position=Angle(0, u.deg), max_position=Angle(360, u.deg),
-                         start_position=Angle(start_position, u.deg), speed=Angle(speed, u.deg))
+        super().__init__(
+            min_position=Angle(0, u.deg),
+            max_position=Angle(360, u.deg),
```

Comment:
again there is a degeneracy at 0 and 360...

Reply:
The actuator in question has no provision for a non-included limit.

Reply:
![image](https://user-images.githubusercontent.com/14132079/74056320-08884f00-499f-11ea-9a61-8736dfaa60c1.png)